hbw/app/javascript/packs/hbw/components/form/file_upload.js.jsx

```diff
@@ -61,15 +62,60 @@ modulejs.define('HBWFormFileUpload', ['React'], (React) => {
 
       const hiddenValue = JSON.stringify({ files });
       const cssClass = cx('hbw-file-upload', params.css_class, { hidden });
-      const errorMessage = env.translator('errors.file_list_field_required');
-      const errorMessageCss = cx('alert', 'alert-danger', { hidden: fileListPresent });
+
+      const errMessageParamAbcent = env.translator(
+        'errors.file_list_name_required',
+        { field_name: name }
+      );
+
+      const errMessageParamEmpty = env.translator(
+        'errors.file_list_name_empty',
+        { field_name: name }
+      );
+
+      const errMessageParamNotMatched = env.translator(
+        'errors.file_list_name_not_matched',
+        { file_list_name: params.file_list_name }
+      );
+
+      const errMessageParamNotUnique = env.translator(
+        'errors.file_list_name_not_unique',
+        { file_list_name: params.file_list_name }
+      );
+
+      const errMessageCss = cx(
+        'alert',
+        'alert-danger'
+      );
+
+      const fieldReferToUnuniqueFileListName = () => (
+        fileListNames.filter(el => params.file_list_name === el).length > 1
+      );
```

Comment:
Name is opposite to content.

Reply:
It renamed to **fieldReferSeveralFileListNames**.